dart/lib/src/sentry_options.dart

```diff
@@ -184,12 +184,29 @@ class SentryOptions {
   /// sent. Events are picked randomly. Default is null (disabled)
   double? sampleRate;
 
+  /// (Web only) Errors only occurring on these Urls will be handled and sent to sentry.
+  /// If an null or an empty list is used, the SDK will send all errors.
+  /// To use regex add the `^` and the `$` to the string.
+  ///
+  /// If used on a platform other than Web, this setting will be ignored.
+  List<String> allowUrls = [];
+
+  /// (Web only) Errors occurring on these Urls will be ignored and are not sent to sentry.
+  /// If an null or an empty list is used, the SDK will send all errors.
+  /// In combination with `allowUrls` you can block subdomains of the domains listed in allowUrls.
+  /// To use regex add the `^` and the `$` to the string.
+  ///
+  /// If used on a platform other than Web, this setting will be ignored.
+  List<String> denyUrls = [];
```

Comment:
I think since these are flutter specific options both options should be moved to `sentry_flutter_options.dart`